lib/rules/max-top-level-suites.js

```diff
@@ -0,0 +1,49 @@
+'use strict';
+
+/**
+ * @fileoverview Limit the number of top-level suites in a single file
+ * @author Alexander Afanasyev
+ */
+
+var R = require('ramda'),
+    astUtil = require('../util/ast'),
+    defaultSuiteLimit = 1;
+
+module.exports = function (context) {
+    var stack = [],
+        topLevelDescribes = [],
+        suiteLimit = context.options[0];
+
+    if (R.isNil(suiteLimit)) {
+        suiteLimit = defaultSuiteLimit;
+    } else {
+        suiteLimit = suiteLimit.limit;
```

Comment:
@jfmengels okay, good idea, added an empty object handling with a corresponding test. 

By the way, when I write it as:

```js
suiteLimit = defaultSuiteLimit ? R.isNil(options) : options.limit
```

I'm getting a lot of `TypeError: Cannot read property 'loc' of undefined` which I think is a bug in ESLint - so, have to use the explicit if/else in this case.

As far as documenting "options" - I think in the link you've posted "settings" are actually described..but in this case `limit` is a rule option..

Thanks.